Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -398,6 +457,18 @@ extension ActivityStreamPanel: DataObserverDelegate {
         }
     }
 
+    func getBookmarks() -> Success {
+        // Try to fetch only a row of bookmarks. On rotation there might be a few more bookmarks than we want but thats okay.
+        let num = Int(Section(1).numberOfItemsForRow())
```

Comment:
This should probably be `Section.bookmarks.numberOfItemsForRow()` for better readability or in case more sections are added down the road.